plansys2_popf_plan_solver/src/plansys2_popf_plan_solver/popf_plan_solver.cpp

```diff
@@ -150,8 +150,9 @@ POPFPlanSolver::is_valid_domain(
 
   std::string result((std::istreambuf_iterator<char>(plan_file)),
     std::istreambuf_iterator<char>());
+  std::transform(result.begin(), result.end(), result.begin(), ::tolower);//result output plan to lower case
 
-  return result.find("Solution Found") != result.npos;
+  return result.find("error") == result.npos;//no "error" string in output from the planner -> domain valid
```

Comment:
I think this will not pass cpplint because spaces around `//`are needed (2 before and 1 after, I think), and also the line seems too long (is it <= 100). Please check it by running `colcon test`

Reply:
I opened another MR for this same issue before realizing that this MR already existed. ... Sorry about that. My solution is a little different from this one, but essentially the same.

I'll leave my MR open in case you want to take a look. Feel free to decline, if you decide to go with this approach.